src/pages/en/install/auto.md

```diff
@@ -127,33 +108,16 @@ If all goes well, Astro should now be serving your project on [http://localhost:
 
 Astro will listen for live file changes in your `src/` directory, so you will not need to restart the server as you make changes during development.
 
-If you aren't able to open your project in the browser, go back to the terminal where you ran the `start` command to see what went wrong.
-
-## 4. Deploy to the web
-
-It's time to deploy your project to the web! Run the `build` command in your project to build your static website to a new `dist/` folder in your project.
-
-```bash
-# npm
-npm run build
-
-# yarn
-yarn build
-
-# pnpm
-pnpm run build
-```
-
-When the command finishes, you should have a new `dist/` folder in your project that you can deploy directly to your favorite web host.
-
-To get started hosting your website for free, check out our proud hosting partner, [Netlify](https://www.netlify.com/). For instructions on deploying to whatever host you choose, read our detailed [deployment guide](/en/guides/deploy/).
+If you aren't able to open your project in the browser, go back to the terminal where you ran the `start` command and look to see if an error occurred, or if your project is being served at a different URL than the one linked to above.
 
 ## Next Steps
 
-Success! Now you're ready to start developing!
+Success! You are now ready to start building with Astro! 🥳
+
+Here are a few topics that we recommend exploring next. You can read them in any order. You can even leave our documentation for a bit and go play in your new Astro project codebase, coming back here whenever you run into trouble or have a question.
 
-📚 Learn more about Astro’s project structure in our [Project Structure guide](/en/core-concepts/project-structure/).
+📚 **Add a framework:** Learn how to extend Astro with support for React, Svelte, Tailwind and more in our [Integrations guide](/en/guides/integrations-guide/).
```

Comment:
```suggestion
📚 **Add a framework:** Learn how to extend Astro with support for React, Svelte, Tailwind and more with `npx astro add` in our [Integrations guide](/en/guides/integrations-guide/).
```
@delucis - Do you think this strikes a good balance? It doesn't exactly SAY run the command... it says "Go learn about the command!" But those who are adventurous/confident can just run the command right away,

Reply:
Don't love the double "with". Maybe 
```suggestion
📚 **Add a framework:** Learn how to extend Astro with support for React, Svelte, Tailwind and more using `npx astro add` in our [Integrations guide](/en/guides/integrations-guide/).
```